src/main/java/uk/gov/hmcts/divorce/document/content/ServiceOrderTemplateContent.java

```diff
@@ -91,14 +91,17 @@ public Map<String, Object> apply(final CaseData caseData,
         templateContent.put(SERVICE_APPLICATION_RECEIVED_DATE,
             alternativeService.getReceivedServiceApplicationDate().format(DATE_TIME_FORMATTER));
         templateContent.put(IS_SERVICE_ORDER_TYPE_DEEMED, DEEMED.equals(alternativeService.getAlternativeServiceType()) ? YES : NO);
-        templateContent.put(DUE_DATE, caseData.getDueDate().format(DATE_TIME_FORMATTER));
 
         if (serviceApplicationDecisionDate != null) {
             templateContent.put(SERVICE_APPLICATION_DECISION_DATE,
                 serviceApplicationDecisionDate.format(DATE_TIME_FORMATTER));
         }
 
-        if (NO.equals(alternativeService.getServiceApplicationGranted())) {
+        if (alternativeService.isApplicationGranted()) {
+            if (caseData.getDueDate() != null) {
+                templateContent.put(DUE_DATE, caseData.getDueDate().format(DATE_TIME_FORMATTER));
```

Comment:
I think this will throw 400 in docmosis as we are not passing due date. May be we should pass empty when due date is not present ?

Reply:
updated